qiime2/sdk/action.py

```diff
@@ -284,6 +282,47 @@ def _build_annotations(self):
 
         return annotations
 
+    def _build_numpydoc(self):
+        numpydoc = ""
+        numpydoc += "{}\n\n".format(self.name)
+        numpydoc += "{}\n\n".format(self.description)
```

Comment:
This is true of all of the descriptions (param, etc), but `textwrap` also understands indenting, which will make things easier.